docs/Tutorials/Tutorial-design.md

```diff
@@ -0,0 +1,38 @@
+# Tutorial Design
+
+The purpose of a hack week is to expose participants to a broad range of data science tools and methods. With this in mind, we suggest the following guidelines when developing a tutorial:

```

Comment:
*A* purpose, maybe? For Astro Hack Week, it's certainly not the only purpose, and if it were, I think I might just organize a summer school instead? :)